liota/dccs/iotcc.py

```diff
@@ -83,6 +90,12 @@ def __init__(self, con):
         self.enable_reboot_getprop = read_liota_config('IOTCC_PATH', 'enable_reboot_getprop')
         self.counter = 0
         self.recv_msg_queue = self.comms.userdata
+        self.req_ops_lock = Lock()
+        self.req_dict = {}
+        thread = threading.Thread(target=self._dispatch_recvd_msg)
+        thread.daemon = True
+        # This thread will continuously run in background to check and dispatch received responses
+        thread.start()
```

Comment:
I was just wondering about the cleanup of this dcc. Once loaded, these threads keep on running forever. Can we do anything to stop them during cleanup step? Of course this applies to the existing thread as well along with this thread and can be handled separate from this pull request in future.

Reply:
Agreed.